.ci/Jenkinsfile

```diff
@@ -392,6 +433,21 @@ def generateStep(Map params = [:]){
   }
 }
 
+def prepareRelease(String nodeVersion='node:lts', Closure body){
```

Comment:
We might need to double check if `node:lts` can be used for the release or it's required some other version.

Reply:
@watson Can you give us insight on this point? What are the requirements for the version of node that we use when running `npm publish`? Is using LTS your recommendation?

Reply:
It doesn't really matter much. LTS is fine 👍

Reply:
Sounds good. We'll stick with that for now. It's easy to change in the future. :) Thanks!